mesa/examples/basic/boltzmann_wealth_model/app.py

```diff
@@ -7,12 +7,8 @@
 
 
 def agent_portrayal(agent):
-    size = 10
-    color = "tab:red"
-    if agent.wealth > 0:
-        size = 50
-        color = "tab:blue"
-    return {"size": size, "color": color}
+    color = agent.wealth
```

Comment:
Maybe add a comment here what’s happening for those not familiar with colormapping.